app/controllers/languages_controller.rb

```diff
@@ -12,16 +13,27 @@ def show
   end
 
   def create
-    language = Language.create!(permit_params(:name, :code, :direction))
+    language = Language.create!(permit_params(:name, :code, :direction, :force_language_name))
     response.headers["Location"] = "languages/#{language.id}"
     render json: language, status: :created
   rescue ActiveRecord::RecordNotUnique
     raise Error::BadRequestError, "Code #{data_attrs[:code]} already exists."
   end
 
+  def update
+    language = load_language
+    language.update!(permit_params(:name, :direction, :force_language_name))
```

Comment:
> This endpoint should not support updating the assigned bcp47 language code

That's a reference to `code` rather than `name`, right?

Reply:
correct, the BCP47 language code is what's stored in the `code` attribute, there are cascade effects that happen when updating the code so we are not allowing it via the admin UI.